recipe/meta.yaml

```diff
@@ -173,17 +165,16 @@ outputs:
         - {{ compiler('c') }}
         - {{ stdlib('c') }}
         - {{ compiler('cxx') }}
-        # 20221027 - hmaarrfk - needs distutils, which was removed from 3.11
-        - cross-python_{{ target_platform }} 3.10    # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
         - pkg-config
         - meson
         - ninja
         - gettext-tools
+        - python <{{ 3.15 }}
       host:
         - libgl-devel                        # [linux]
         - libegl-devel                       # [linux]
         - libdrm                             # [linux]
-        - python 3.10  # [build_platform != target_platform]
```

Comment:
just leave python, but you can make it 3.12 if you like.

Reply:
Does it need to be there at all? I removed that line altogether, and things are building.

Reply:
Or is that supposed to be under hosts: ?

Reply:
sure that works too.

i mean, the comment specifed why 3.10 was there.

If that isn't true anymore, you can adapt things as needed.

Reply:
I seem to remember that before the last rerender, the build didn't run all 4 versions of python.  Maybe I need to rerender again?